src/interceptors/fetch/index.ts

```diff
@@ -147,17 +153,15 @@ export class FetchInterceptor extends Interceptor<HttpRequestEventMap> {
           requestId,
         })
 
-        const response = new Response(mockedResponse.body, mockedResponse)
```

Comment:
I removed the cloning altogether, I don't think it's needed here. We can use the original mocked response instance returned from the listener and just forward it to the consumer.